### PR TITLE
Update/add hayabusa modules

### PR DESCRIPTION
**Pull request by gs3cl**
## Description

Please include a summary of the change and (if applicable) which issue is fixed.

## Checklist:
Please replace every instance of `[ ]` with `[X]` OR click on the checkboxes after you submit your PR

- [x] I have generated a unique `GUID` for my Target(s)/Module(s)
- [x] I have placed the Target(s)/Module(s) in an appropriate subfolder in Targets or Modules. If one doesn't exist, I have either added it to the `Misc` folder or created a relevant subfolder **with justification**
- [x] I have set or updated the version of my Target(s)/Module(s)
- [x] I have verified that KAPE parses the Target(s)/Module(s) successfully via kape.exe, using `--tlist`/`--mlist` and corrected any errors 
- [x] I have validated my Target(s)/Module(s) against test data and verified they are working as intended
- [x] I have made an attempt to document the artifacts within the Target(s) or Module(s) I am submitting. If documentation doesn't exist, I have placed `N/A` underneath the Documentation header

Thank you for your submission and for contributing to the DFIR community!

````
System info: Machine name: DESKTOP, 64-bit: True, User: KAPE OS: Windows10 (10.0.19044)

Using Module operations
        Flushing module destination directory 'C:\temp\0_Hayabusa'
        Creating module destination directory 'C:\temp\0_Hayabusa'
        Module 'hayabusa_LiveResponse': Found 1 processor
                Found processor 'Executable: hayabusa\hayabusa.exe, Cmd line: --live-analysis -m med --European-time --quiet -U -o %destinationDirectory%\hayabusa.csv, Export: csv, Append: False'!
        Module 'hayabusa_logon': Found 1 processor
                Found processor 'Executable: hayabusa\hayabusa.exe, Cmd line: --live-analysis --European-time --quiet -U --logon-summary, Export: csv, Append: False'!
        Module 'hayabusa_OfflineEventLogs': Found 1 processor
                Found processor 'Executable: hayabusa\hayabusa.exe, Cmd line: -d %sourceDirectory% --European-time --quiet -U -o %destinationDirectory%\hayabusa.csv, Export: csv, Append: False'!
        Module 'hayabusa_statistics': Found 1 processor
                Found processor 'Executable: hayabusa\hayabusa.exe, Cmd line: --live-analysis -s, Export: csv, Append: False'!
Discovered 4 processors to run.
````

### CODE REVIEW

**Comment by AndrewRathbun** [comment]
@gs3cl can you try to fix what the linter is hitting on? I'm not going to get to the computer until a bit later.

File: Modules/Apps/GitHub/hayabusa_logon.mkape [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+Version: 1.0
+Id: 2dd36453-d175-4b54-b98f-9cdf492859ce
+ExportFormat: csv
+Processors:
+    -
+        Executable: hayabusa\hayabusa.exe
+        CommandLine: --live-analysis --European-time --quiet -U --logon-summary
+        ExportFormat: csv
+        ExportFile: hayabusa_logon.csv
+# Documentation
```
Comment (AndrewRathbun) [inline Modules/Apps/GitHub/hayabusa_logon.mkape:13]: Need a new line above this one

File: Modules/Apps/GitHub/hayabusa_statistics.mkape [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+Version: 1.0
+Id: f3fb3648-1ccb-458c-80e2-261818803bda
+ExportFormat: csv
+Processors:
+    -
+        Executable: hayabusa\hayabusa.exe
+        CommandLine: --live-analysis -s
+        ExportFormat: csv
+        ExportFile: hayabusa_statistics.csv
+# Documentation
```
Comment (AndrewRathbun) [inline Modules/Apps/GitHub/hayabusa_statistics.mkape:13]: Need a new line above this one

**Comment by gs3cl** [comment]
should be fine now

**Review by AndrewRathbun — APPROVED** [review]

**Pull request merged**